requirements.txt

```diff
@@ -1,5 +1,6 @@
 ansible==2.1.1.0
+GDAL==2.1.0
```

Comment:
this will require gdal-dev (on debian for example) installed on user's system. On Ubuntu there is an issue where we cannot use gdal 2.1.0 because of conflict with other packages.